milestone1.css

```diff
@@ -0,0 +1,139 @@
+@import url('https://fonts.googleapis.com/css2?family=Open+Sans&display=swap');
+
+*{
+    list-style: none;
+    text-decoration: none;
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+    font-family: 'Open Sans', sans-serif;
+}
+
+body{
+    background: #ffffff;
+}
+
+.wrapper .sidebar{
+    background: rgb(5, 68, 104);
+    position: fixed;
+    top: 0;
+    left: 0;
+    width: 225px;
+    height: 100%;
+    padding: 20px 0;
+    transition: all 0.5s ease;
+}
+.wrapper .sidebar .profile{

```

Comment:
in your css,  It is redundant to include .wrapper .sidebar, since your profile is nested inside of these classes in your html.  

so in the following code, any styles that wrapper class has will exist in the sidebar and profile by default, as they exist inside of the wrapper class. 
```
<div class="wrapper"> 
    <div class="sidebar">
        <div class="profile">
        </div>
    </div>
</div>
```

what you are doing works, but you can save some time by removing redundant code!